src/logic/operate.js

```diff
@@ -0,0 +1,33 @@
+/* eslint-disable import/extensions */
+import Big from 'big.js';
+
+function operate({ numberOne, numberTwo, operation }) {
+  let total;
+  const opertionCalc = ['+', '-', 'x', '÷', '%'];
+
+  switch (operation) {
+    case opertionCalc[0]:
+      total = Big(numberOne).plus(numberTwo);
+      break;
+    case opertionCalc[1]:
+      total = Big(numberOne).minus(numberTwo);
+      break;
+    case opertionCalc[2]:
+      total = Big(numberOne).times(numberTwo);
+      break;
+    case opertionCalc[3]:
+      total = Big(numberOne).div(numberTwo);
+      break;
+    case opertionCalc[4]:
+      total = Big(numberOne).times(numberTwo) / 100;
```

Comment:
This should deal with just the first number ![see this](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/react_reducx/calculator/milestone4/performOperate.png)